src/main/kotlin/no/uio/microobject/runtime/REPL.kt

```diff
@@ -39,7 +39,7 @@ class Command(
 
 @Suppress("DEPRECATION") // ReasonerFactory is deprecated by HermiT but I keep it like this to make a change easier
 class REPL(private val settings: Settings) {
-    private var interpreter: Interpreter? = null
+    public var interpreter: Interpreter? = null
```

Comment:
Can you briefly say why this is necessary?  The  commit message doesn't tell

Reply:
Eduard asked me to put it public to be able to run queries on the program state via the `Interpreter` object.
I noticed later that actually there is a getter `getInterpreter()` (which I used) but, since I have no experience with Kotlin, I'm not sure if it has been generated after I set the variable public or it already existed (I haven't seen any java-like annotation that could have created it by default)